src/pybind/mgr/dashboard/controllers/host.py

```diff
@@ -1,12 +1,30 @@
 # -*- coding: utf-8 -*-
 from __future__ import absolute_import
 
-from . import ApiController, AuthRequired, RESTController
+import json
+import cherrypy
+
+from mgr_module import CommandResult
+from . import ApiController, AuthRequired, RESTController, Endpoint
 from .. import mgr
+from ..services.ceph_service import CephService
 
 
 @ApiController('/host')
 @AuthRequired()
 class Host(RESTController):
     def list(self):
         return mgr.list_servers()
+
+    @Endpoint('PUT')
```

Comment:
Why not using POST?

Reply:
We are using POST on creation operations. 
Since this is not creating anything I tough it would be more adequate to use PUT.

Reply:
I would favor using POST when we are implementing an "action" as is this case. For me PUT only makes sense when updating REST resources. What do others (@votdev @ricardoasmarques @sebastian-philipp) think about this?

Reply:
I would favor `PUT` for idempotent operations. In this case, calling this endpoint twice is not the same as calling it only one time, so I guess you shoud use a `POST`.